org.eclipse.wildwebdeveloper.xml/pom.xml

```diff
@@ -4,10 +4,10 @@
 	<parent>
 		<groupId>org.eclipse.wildwebdeveloper</groupId>
 		<artifactId>parent</artifactId>
-		<version>1.0.0-SNAPSHOT</version>
+		<version>1.0.1-SNAPSHOT</version>
```

Comment:
Please, roll it back to be `<version>1.0.0-SNAPSHOT</version>` - we haven't change parent version yet